src/driver/tools/configure_kernel.sh

```diff
@@ -21,7 +21,7 @@ fi
 
 # ---- Output header path ------------------------------------------------
 
-OUT="driver/amdxdna/config_kernel.h"
+OUT="${OUT:-driver/amdxdna/config_kernel.h}"
```

Comment:
Why making this change? The OUT needs to be set to this value. Otherwise, it won't work.

Reply:
OK, sounds good, then.